test/Fondue.test.js

```diff
@@ -372,11 +391,15 @@ describe("Layout features", () => {
 
 		expect(fondue.featureChars["latn"]["dflt"]).toEqual(
 			expect.objectContaining({
-				liga: {
-					alternateCount: [],
-					input: ["ffl", "ffi", "ff", "fl", "fi"],
-					type: 4,
-				},
+				liga: [
+					{
+						alternateCount: [],
+						lookahead: [],
+						backtrack: [],
```

Comment:
Could you add a test where these values are not empty? I guess the code you've added is not really exercised in tests now, since all these tests have empty `lookahead` and `backtrack` values.

Reply:
Will add, thanks! 👍

Reply:
Done!

Also added checks for the new `summary` thing.